terraform-plan/action.yaml

```diff
@@ -36,3 +39,5 @@ runs:
     - ${{ inputs.score-threshold }}
     - --log-level
     - ${{ inputs.log-level }}
+  env:
+    MONDOO_CONFIG_BASE64: ${{ inputs.service-account-credentials }}
```

Comment:
Isn't that overriding the env variable with an empty value when the user sets:

```
env:
          MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
```

Reply:
No, it looks like the env var set when calling the action gets precedence over the env var I pre-set here.

I included the env var case and the args case in this test: https://github.com/mondoohq/actions/blob/5bb92b8e6b440403349d39f074e387b780b194b8/.github/workflows/docker-image.yaml

The tests look good:
https://github.com/mondoohq/actions/actions/runs/3713922613/jobs/6297163046